cli/stork-publisher-agent/processor_test.go

```diff
@@ -113,3 +113,107 @@ func TestDeltaOnly(t *testing.T) {
 	assert.Equal(t, int64(30000000), nextResult[assetId].SignedPrice.TimestampedSignature.Timestamp)
 	assert.Equal(t, QuantizedPrice("2000000000000000000"), nextResult[assetId].SignedPrice.QuantizedPrice)
 }
+
+func TestZeroPrice(t *testing.T) {
+	logger := zerolog.New(os.Stdout).With().Timestamp().Logger()
+	config := NewStorkPublisherAgentConfig(
+		[]SignatureType{EvmSignatureType},
+		evmPrivateKey,
+		evmPublicKey,
+		starkPrivateKey,
+		starkPublicKey,
+		time.Duration(0),
+		10*time.Millisecond,
+		DefaultChangeThresholdPercent,
+		"czowx",
+		DefaultStorkRegistryBaseUrl,
+		time.Duration(0),
+		time.Duration(0),
+		storkAuth,
+		"",
+		"",
+		"",
+		time.Duration(0),
+		time.Duration(0),
+		false,
+		0,
+	)
+
+	signer, err := NewSigner[*EvmSignature](*config, EvmSignatureType, logger)
+	if err != nil {
+		t.Fatalf("NewSigner[*EvmSignature]: %v", err)
+	}
+
+	inputCh := make(chan ValueUpdate)
+	outputCh := make(chan SignedPriceUpdateBatch[*EvmSignature])
+	processor := NewPriceUpdateProcessor[*EvmSignature](
+		*signer,
+		1,
+		config.ClockPeriod,
+		config.DeltaCheckPeriod,
+		DefaultChangeThresholdPercent,
+		false,
+		inputCh,
+		outputCh,
+		logger,
+	)
+
+	go processor.Run()
+
+	// initial zero update gets sent out
```

Comment:
1. initial zero price update triggers the delta
2. subsequent zero price update does not
3. nonzero price update triggers the delta
4. returning to zero triggers the delta